rosbag2_storage/include/rosbag2_storage/storage_options.hpp

```diff
@@ -56,6 +57,10 @@ struct StorageOptions
   // Defaults to disabled.
   bool snapshot_mode = false;
 
+  // Start and end time for cutting
+  rcutils_time_point_value_t start_time = 0;
+  rcutils_time_point_value_t end_time = 0;
+
```

Comment:
To avoid bringing extra dependencies I would recommend using a plain `uint64_t` type for datestamps or durations similar to what we already use for the `max_bagfile_duration`

Reply:
In rosbag2_transport/include/rosbag2_transport/play_options.hpp the datatype rcutils_time_point_value_t is used for start_offset (line 87):

```
  // Time to start playback as an offset from the beginning of the bag.
  rcutils_time_point_value_t start_offset = 0;
```

I used this as guideline. Do you think this would be better of using uint64_t also (IMHO this should be same type everywhere)?

Reply:
@pfavr2 The difference is that `start_offset` lives only on the `rosbag2_transport` and upper layers.
As have said we would like to avoid extra dependencies on the bottom layers and already using `uint64_t` type for time duration. 
For consistency, it would be more reasonable to use the same `uint64_t` type for the `start` and `end`  timestamps. Since they are in nanoseconds it shouldn't cause any problems or difficulties.